services/importer/lib/importer/runner.rb

```diff
@@ -277,8 +278,43 @@ def single_resource_import
 
               log.store   # Checkpoint-save
               log.append "Filename: #{source_file.fullpath} Size (bytes): #{source_file.size}"
-              import_stats = execute_import(source_file, @downloader)
-              @stats << import_stats
+
+              # Hack for Samples 2.0 Save As - Tech debt to clean up with more elegant solution
+              if(source_file.extension.split('.').last == 'gpkg')
```

Comment:
what happens if the file doesn't have an extension?

Reply:
should we use a different file extension for the save as? that way it's clear that it's not a standard package.

Reply:
Yeah, we should check for `.carto.gpkg` explicitly. That extension is unique to us